test/functional/test_runner.py

```diff
@@ -149,8 +149,10 @@
     'tool_wallet.py --descriptors',
```

Comment:
I don't quite follow. Why would this change whether it runs sequentially or parallel?

Reply:
My understanding is that a missing type setting will run both types, if possible, sequentially.

The goal of test_runner is to run as much in parallel as possible. So if the test_runner setting is missing a wallet type setting in one line, it will run the test twice, sequentially.

This can be fixed by forcing a type in the test runner, thus running at most one test per line, thus all lines/tests in parallel.

Reply:
Done.

There are a few tests that do `add_wallet_options` but don't really test anything with the wallet. For those tests, I've added an `any_type` to `add_wallet_options` so that those tests don't need either `--legacy-wallet` or `--descriptors` specified and won't be run twice by the test runner.